### ISSUE TITLE
Change Module Format To Tabs

### ISSUE THREAD

**Issue by burkeholland** [NativeScript/docs-v7#313]
Sencha's docs have [a really nice tabbed layout](http://docs.sencha.com/extjs/4.2.2/#!/api/Ext.picker.Date) for their widgets that contains all of the properties, methods and events on the same page. No jumping around. Can we consider implementing a format similar to this?

**Comment by tjvantoll** [comment]
Agreed that this is far easier to navigate than https://docs.nativescript.org/api-reference/globals.html. Something to consider if we ever work on the design of the API docs.
